code/game/mecha/mecha_construction_paths.dm

```diff
@@ -930,22 +930,95 @@
 		playsound(holder, 'sound/items/bikehorn.ogg', 50, 1)
 		user.visible_message("HONK!")
 
+/datum/construction/unordered/mecha_chassis/reticence
+	result = /datum/construction/mecha/reticence
+	steps = list(
+		/obj/item/mecha_parts/part/reticence_torso,
+		/obj/item/mecha_parts/part/reticence_left_arm,
+		/obj/item/mecha_parts/part/reticence_right_arm,
+		/obj/item/mecha_parts/part/reticence_left_leg,
+		/obj/item/mecha_parts/part/reticence_right_leg,
+		/obj/item/mecha_parts/part/reticence_head
+	)
+
+
+/datum/construction/mecha/reticence
+	result = /obj/mecha/combat/reticence
+	steps = list(
+		//1
+		list(
+			"key" = /obj/item/circuitboard/mecha/reticence/main,
+			"action" = ITEM_DELETE
+		),
+
+		//2
+		list(
+			"key" = /obj/item/circuitboard/mecha/reticence/peripherals,
+			"action" = ITEM_DELETE
+		),
+
+		//3
+		list(
+			"key" = /obj/item/circuitboard/mecha/reticence/targeting,
+			"action" = ITEM_DELETE
+		),
+
+		//4
+		list(
+			"key" = /obj/item/stock_parts/cell,
+			"action" = ITEM_MOVE_INSIDE
+		),
+
+		//5
+		list(
+			"key" = /obj/item/clothing/suit/suspenders,
+			"action" = ITEM_DELETE
+		),
+
+		//6
+		list(
+			"key" = /obj/item/clothing/mask/gas/mime,
+			"action" = ITEM_DELETE
+		),
+
+		//7
+		list(
+			"key" = /obj/item/clothing/head/beret,
+			"action" = ITEM_DELETE
+		),
+	)
+
+// Reticence doesn't have any construction step icons, so we just set an icon once.
+/datum/construction/mecha/reticence/update_holder(step_index)
+	..()
+	holder.icon = 'icons/mecha/mecha.dmi'
+	holder.icon_state = "reticence" //oh like anyone is going to be able to tell. Blame paradise for not having the sprites.

```

Comment:
I'm scared to ask what this comment means.

Reply:
Paradise doesn't have an analogue of the normal mech construction sprites (wires, etc), and whatever they do use, I can't decipher it, or translate it into our system.

Reply:
So, i just set to the sprite to be a fully constructed reticence mech every construction step, since its 95% transparent you can't tell anyway.

Reply:
oh ok lol